unix/lib/time.mli

```diff
@@ -18,6 +18,6 @@ type +'a io = 'a Lwt.t
 
 (** Timeout operations. *)
 
-val sleep : float -> unit Lwt.t
```

Comment:
@talex5 what is the purpose of keeping the function?  my impression was that mirage-unix/xen is an implementation of (some of the modules defined in) mirage-types...